### PR TITLE
Refine locale setup again

### PR DESCRIPTION
**Pull request by Vogtinator**
Use live-langset-data.

This required a rewiring of the jeos-firstboot.service so I also changed the way it's enabled to fix a different bug.

### CODE REVIEW

File: files/usr/lib/systemd/system/jeos-firstboot.service [hunk 1]
```diff
@@ -7,19 +7,25 @@
...
+Before=network.service systemd-user-sessions.service
 
 [Service]
-Environment=SYSTEMCTL_OPTIONS=--ignore-dependencies TERM=linux
 Type=oneshot
+Environment=SYSTEMCTL_OPTIONS=--ignore-dependencies TERM=linux
 RemainAfterExit=yes
+ExecStartPre=-/usr/bin/plymouth quit
 ExecStart=/usr/lib/jeos-firstboot
+ExecStartPost=/usr/bin/rm -f /var/lib/YaST2/reconfig_system
```
Comment (mnowaksuse) [inline files/usr/lib/systemd/system/jeos-firstboot.service]: Are we going to have this file? (again)

File: files/usr/lib/systemd/system/jeos-firstboot.service [hunk 2]
```diff
@@ -7,19 +7,25 @@
...
+
+# jeos-firstboot starts before network and login though
+Before=network.service systemd-user-sessions.service
 
 [Service]
-Environment=SYSTEMCTL_OPTIONS=--ignore-dependencies TERM=linux
 Type=oneshot
+Environment=SYSTEMCTL_OPTIONS=--ignore-dependencies TERM=linux
 RemainAfterExit=yes
+ExecStartPre=-/usr/bin/plymouth quit
```
Comment (mnowaksuse) [inline files/usr/lib/systemd/system/jeos-firstboot.service]: Do we have `plymouth`?

File: files/usr/lib/systemd/system/jeos-firstboot.service [hunk 3]
```diff
@@ -7,19 +7,25 @@
...
-ConditionPathIsReadWrite=/etc
-RequiresMountsFor=/.snapshots
-ConditionFirstBoot=yes
+
+# Same as YaST2-Firstboot.service here
+After=apparmor.service local-fs.target plymouth-start.service YaST2-Second-Stage.service
+Conflicts=plymouth-start.service
+Before=getty@tty1.service serial-getty@ttyS0.service serial-getty@ttyS1.service serial-getty@ttyS2.service
+Before=display-manager.service
+ConditionPathExists=/var/lib/YaST2/reconfig_system
```
[the remark this answers is not in the document]
Comment (Vogtinator) [reply files/usr/lib/systemd/system/jeos-firstboot.service]: Yes, it's added by the image build.

File: files/usr/lib/systemd/system/jeos-firstboot.service (hunk 2 above)
Comment (Vogtinator) [reply files/usr/lib/systemd/system/jeos-firstboot.service]: Not on JeOS for SLE 12/15, but that might change in the future and/or be different for openSUSE - calling plymouth quit can't hurt here (the "-" means that it won't fail).

**Pull request merged**